packages/lib-user/src/components/UserHome/UserHome.js

```diff
@@ -19,6 +20,12 @@ function UserHome({ authUser }) {
         <MyGroupsContainer previewLayout authUser={authUser} login={authUser.login} />
       </Grid>
       <RecentSubjectsContainer authUser={authUser} />
+      <Box pad={size !== 'small' ? '0' : { horizontal: '30px' }}>
```

Comment:
I noticed during this PR that lib-user's Layout does not handle overall PageBody padding. Instead, the ContentBox components individually change their padding based on screen size 🤔 There could be some unification of PageBody padding Layout